flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/planner/delegation/hive/HiveParserUtils.java

```diff
@@ -155,6 +153,11 @@ public class HiveParserUtils {
 
     private static final Logger LOG = LoggerFactory.getLogger(HiveParserUtils.class);
 
+    public static final String BRIDGING_SQL_FUNCTION_CLZ_NAME =
+            "org.apache.flink.table.planner.functions.bridging.BridgingSqlFunction";
+    private static final String bridgingSqlAggFunctionClzName =
```

Comment:
`bridgingSqlAggFunctionClzName` should be `BRIDGING_SQL_AGG_FUNCTION_CLZ_NAME`

Reply:
Since it is private, it should be `bridgingSqlAggFunctionClzName`.